src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,33 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int NUMBER_ONE = 1;
+    private static final int NUMBER_TWO = 2;
+    private static final int NUMBER_THREE = 3;
```

Comment:
```suggestion
    private static final int NAME_INDEX = 1;
```

NUMBER_ONE, NUMBER_TWO, ... not informative constant names